packages/a-msgpack/src/neat/extensions.ts

```diff
@@ -19,17 +19,16 @@ import { ExtensionCodec } from '../ExtensionCodec';
 import { decode } from '../decode';
 import { encode } from '../encode';
 
-import { Pointer } from './NeatTypes';
+import { Pointer, Wildcard } from './NeatTypes';
 
-type PointerEncoder = (data: unknown) => Uint8Array | null;
+type PointerEncoder = (data: Pointer) => Uint8Array;
 type PointerDecoder = (buffer: Uint8Array) => Pointer;
+type WildcardEncoder = (data: unknown) => null;
```

Comment:
why not `Wildcard` instead of `unknown`?

Reply:
Yeah this is actually wrong. I'm surprise ts didn't catch this. You shouldn't be able to pass any data to the wildcard encoder